data/countries/de/sh/rendsburg-eckernfoerde/data.yaml

```
@@ -64,6 +64,16 @@ urls:
   - type: WEBSITE
     url: https://gruene-felde.de/
 ---
+city: Fleckeby
+country: DE
+district: Rendsburg-Eckernförde
+level: DE:ORTSVERBAND
+state: Schleswig-Holstein
+type: REGIONAL_CHAPTER
+urls:
+  - type: WEBSITE
+    url: https://www.gruene-fleckeby.de/
```

```suggestion
    url: https://gruene-fleckeby.de/
```

www. redirects to https://gruene-fleckeby.de/